### PR TITLE
Adding arguments for replacing instances

### PR DESCRIPTION
**Pull request by hastimr**
Also added a quick update to README

### CODE REVIEW

File: tasks/asg_ec2.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@
     max_size: "{{ aws_asg_max_size | default (1) }}"
     desired_capacity: "{{ aws_asg_desired_capacity | default(1) }}"
     region: "{{ region }}"
+    replace_all_instances: "{{ aws_asg_replace_all_instances }}"
```
Comment (gianrubio) [inline tasks/asg_ec2.yaml]: It would be better if you change the default value to None
``` | default(None)```

File: tasks/asg_ec2.yaml [hunk 2]
```diff
@@ -9,6 +9,8 @@
...
+    replace_batch_size: "{{ aws_asg_replace_batch_size | default(1) }}"
```
Comment (gianrubio) [inline tasks/asg_ec2.yaml]: Same as above, change default for None

File: tasks/asg_elb.yaml [hunk 3]
```diff
@@ -10,6 +10,8 @@
     max_size: "{{ aws_asg_max_size | default (1) }}"
     desired_capacity: "{{ aws_asg_desired_capacity | default(1) }}"
     region: "{{ region }}"
+    replace_all_instances: "{{ aws_asg_replace_all_instances | default(no) }}"
```
Comment (gianrubio) [inline tasks/asg_elb.yaml]: Same as above

File: tasks/asg_elb.yaml [hunk 4]
```diff
@@ -10,6 +10,8 @@
...
+    replace_batch_size: "{{ aws_asg_replace_batch_size | default(1) }}"
```
Comment (gianrubio) [inline tasks/asg_elb.yaml]: Same as above

**Comment by hastimr** [comment]
Thanks for the feedback - updated those variables all to default to None

**Comment by gianrubio** [comment]
@hastimr merged,  you should use version 1.2.5

**Pull request merged**